pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -203,6 +203,52 @@ func NewPolicyApplier(rootNamespace string,
 	}
 }
 
+type RemoteJwksMode int
+
+const (
+	// Istiod is used to indicate Istiod ALWAYS fetches the JWKs server
+	Istiod RemoteJwksMode = iota
+
+	// Hybrid is used to indicate Envoy fetches the JWKs server when there is a cluster entry,
+	// otherwise fallback to Istiod
+	Hybrid
+
+	// Envoy is used to indicate Envoy ALWAYS fetches the JWKs server
+	Envoy
+)
+
+// String converts RemoteJwksMode to readable string.
+func (mode RemoteJwksMode) String() string {
+	switch mode {
+	case Istiod:
+		return "Istiod"
+	case Hybrid:
+		return "Hybrid"
+	case Envoy:
+		return "Envoy"
+	default:
+		return "Unset"
+	}
+}
+
+// convertToRemoteJwksMode converts from string value mode to enum RemoteJwksMode value
+func convertToRemoteJwksMode(mode string) RemoteJwksMode {
+	switch mode {
+	case "istiod":
+		return Istiod
+	case "false":
```

Comment:
Can you comment to indicate its for backwards compat?

btw, you can do `case "istiod", "false":`

Reply:
will do